code/modules/clothing/head/helmet.dm

```diff
@@ -31,7 +31,7 @@
 	desc = "A bulletproof combat helmet that excels in protecting the wearer against traditional projectile weaponry and explosives to a minor extent."
 	icon_state = "helmetalt"
 	item_state = "helmetalt"
-	armor = list(melee = 15, bullet = 40, laser = 10, energy = 10, bomb = 40, bio = 0, rad = 0)
+	armor = list(melee = 15, bullet = 45, laser = 10, energy = 10, bomb = 40, bio = 0, rad = 0)

```

Comment:
This value should be a LOT higher. A "bulletproof combat helmet" should actually protect against bullets! These pathetic things would only cost the enemy 2-3 more bullets (assuming a burst weapon) to down you instead of making him consider another weapon.